### ISSUE TITLE
[booth] `pcs booth ticket add TICKET`does not verify syntactical validity of TICKET

### ISSUE THREAD

**Issue by jnpkrn** [ClusterLabs/pcs#229]
```
# pcs booth ticket add ms_ip_ticket 
# echo $?
> 0

# booth list                                               
> Dec 10 16:03:21 virt-154 booth: [22096]: error: ticket name "ms_ip_ticket" invalid; only alphanumeric names.
> Dec 10 16:03:21 virt-154 booth: [22096]: error:  in config file line 5
> Dec 10 16:03:21 virt-154 booth: [22096]: error: cannot read config
```

That is, pcs will happily allow for (self-inflicted) DoS, which is
apparently bad enough.

Reference:
https://github.com/ClusterLabs/booth/blob/v1.0/src/config.c#L224

There are other checks to be found around:

- is not the ticket name reused (check already present in pcs,
  it seems)

- is not the name of the ticket too long? (BOOTH_NAME_LEN-1 = 63 bytes)

**Comment by jnpkrn** [comment]
As an exception, dash `-` and slash `/` are tolerated, as can be found
in the code.

**Comment by jnpkrn** [comment]
Consequently, the same check shall be applied at
`pcs constraint ticket add TICKET RES`.

**Comment by tomjelinek** [comment]
Ack, tracked at https://bugzilla.redhat.com/show_bug.cgi?id=1791661

**Comment by tomjelinek** [comment]
resolved in #466 and #468
